frontend/src/index.html

```diff
@@ -126,7 +127,7 @@
         </div>
         <div id="hits" class="overflow-x-hidden">
           <template id="item-template">
-            <article class="flex flex-col mb-4 p-1 w-full bg-white rounded-sm shadow-md sm:flex-row sm:p-4" data-file-id="{{objectID}}">
+            <article class="flex flex-col mb-4 p-1 w-full bg-white rounded-sm shadow-md sm:flex-row sm:p-4" x-data="{is_open:false}" data-file-id="{{objectID}}">
```

Comment:
Let's keep the directives content (JS expressions) formatted. I've tried to look for an automatic way, similar to what Prettier does for separate JS files, but could not find such a tool for Alpine yet. Therefore let's maintain proper formatting manually as of now.

I cannot explain all the formatting conventions here, but essentially I'd expect us to use the same formatting as Prettier enforces. You can paste your code into a separate JS file to see how Prettier would format it. Thus gradually you will get accustomed to the conventions.

Reply:
Also, let's follow JS naming conventions, i.e. `isOpen`.